services/minespace-web/src/index.js

```diff
@@ -1,56 +1,114 @@
 import "core-js/stable";
 import "regenerator-runtime/runtime";
 
-import React, { Component } from "react";
+import React, { useState } from "react";
 import { render } from "react-dom";
 import { Provider } from "react-redux";
 
-import { MatomoProvider, createInstance } from "@datapunt/matomo-tracker-react";
-import { ENVIRONMENT } from "@mds/common";
-import { ReactKeycloakProvider} from "@react-keycloak/web";
+import { ReactKeycloakProvider } from "@react-keycloak/web";
+import { useIdleTimer } from "react-idle-timer";
 import App from "./App";
 import "antd/dist/antd.less";
 import "./styles/index.scss";
 import fetchEnv from "./fetchEnv";
 import configureStore from "./store/configureStore";
-import keycloak, {keycloakInitConfig} from "./keycloak";
+import keycloak, { keycloakInitConfig } from "./keycloak";
+import { unAuthenticateUser } from "./actionCreators/authenticationActionCreator";
 
+// eslint-disable-next-line import/prefer-default-export
 export const store = configureStore();
 
-let instance = {};
-
-export class Index extends Component {
-  constructor() {
-    super();
-    this.state = { environment: false };
-    fetchEnv().then(() => {
-      if (ENVIRONMENT.matomoUrl) {
-        instance = createInstance({
-          urlBase: ENVIRONMENT.matomoUrl,
-          enableLinkTracking: false,
-          siteId: 2,
-        });
-      }
-      this.setState({ environment: true });
-    });
-  }
-  
-
-  render() {
-    console.log(keycloakInitConfig, 'jjjj');
-    if (this.state.environment) {
-      return (
-        <ReactKeycloakProvider authClient={keycloak} initOptions={keycloakInitConfig}>
-          <Provider store={store}>
-            <MatomoProvider value={instance}>
-              <App />
-            </MatomoProvider>
-          </Provider>
-        </ReactKeycloakProvider>
-      );
+// 60 seconds before user is inactive- across tabs
+const idleTimeout = 60_000;
```

Comment:
We should bring this timing to Rebecca.  I'm guessing we'll want to be a bit more generous with the timeout.